Apps/Examples/Examples/All Examples/Custom3DPuckExample.swift

```diff
@@ -27,13 +27,13 @@ public class Custom3DPuckExample: UIViewController, ExampleProtocol {
     internal func setupExample() {
 
         // Fetch the `gltf` asset
-        let uri = Bundle.main.url(forResource: "race_car_model",
-                                  withExtension: "gltf")
+        let uri = Bundle.main.url(forResource: "sportcar",
+                                  withExtension: "glb")
 
         // Instantiate the model
         let myModel = Model(uri: uri,
-                            position: [-177.150925, 39.085006],
-                            orientation: [0, 0, 0])
+                            position: [-77.150925, 39.085006],
+                            orientation: [0, 0, 180])
```

Comment:
@ZiZasaurus did you verify puck is heading in the correct direction? On Android we had `[0, 0, 90]` for previous model, a bit surprised to see `[0, 0, 0]` being present on iOS beforehand but let's make sure new model is looking in the correct direction

Reply:
Yes, for this model `[0, 0, 180]` provides the correct heading direction.